tools/goctl/model/sql/template/template.go

```diff
@@ -0,0 +1,128 @@
+package template
+
+import (
+	_ "embed"
+	"fmt"
+	"github.com/zeromicro/go-zero/tools/goctl/util"
```

Comment:
A blank line to separate std package and third party packages.

Reply:
The code has been modified as required